games-strategy/freeorion/freeorion-0.4.10.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{3_6,3_7,3_8} )
+inherit cmake python-single-r1 xdg
+
+DESCRIPTION="A free turn-based space empire and galactic conquest game"
+HOMEPAGE="https://www.freeorion.org"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/freeorion/freeorion.git"
+else
+	if [[ ${PV} = *_p* ]]; then
```

Comment:
I think this was a hack specifically for `0.4.9_p…` because the commit is hardcoded here.